pkgs/development/python-modules/color-manager/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  tqdm,
+  basic-colormath,
+  unstableGitUpdater,
+  setuptools,
+}:
+buildPythonPackage {
+  pname = "color-manager";
+  version = "0-unstable-2024-11-15";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "NicklasVraa";
+    repo = "Color-manager";
+    rev = "9e55e0971ecd0e3141ed5d7d9a8377f7052cef96";
+    hash = "sha256-kXRjp1sFgSiIQC9+fUQcNRK990Hd5nZwJpRGB7qRYrY=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    tqdm
+    basic-colormath
+  ];
+
```

```suggestion

  doCheck = false;

```
I don't think we can test https://github.com/ShayHill/basic_colormath/tree/main/tests

---

I looked into it and I found a way to run the tests using `pytestCheckHook`.